src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -77,44 +96,78 @@ const ButtonWrapper = styled.div`
   align-items: center;
 
   @media (max-width: ${(props) => props.theme.breakpoints.md}) {
-    position: sticky;
+    position: fixed;
     bottom: 0;
+    left: 0;
+    right: 0;
+    width: 100vw;
+    box-sizing: border-box;
```

Comment:
Why do we need this?

Reply:
I removed that, it was I think to include padding in the width but it works ok now.